packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain/SourceNetworkBox.tsx

```diff
@@ -68,21 +68,18 @@ export function SourceNetworkBox({
 
   const isMaxAmount = amount === AmountQueryParamEnum.MAX
 
-  // whenever the user changes the `amount` input, it should update the amount in browser query params as well
```

Comment:
the comment was wrong

but you can't remove this snippet

try https://arbitrum-token-bridge-git-batch-erc20-eth-4-offchain-labs.vercel.app/?amount=max&destinationChain=arbitrum-sepolia&sourceChain=rari-testnet

Reply:
reverted